server/src/ingredient/index.js

```diff
@@ -0,0 +1,9 @@
+import getIngredients from './ingredient.controller';
```

Comment:
this is not how to import from an object export https://www.digitalocean.com/community/tutorials/js-modules-es6